Makefile

```diff
@@ -33,7 +33,7 @@ bin/buildout: venv/bin/python
 	$(VENV)/bin/python bootstrap.py
 
 ve/ venv/bin/python venv/bin/pip:
-	virtualenv -p python2 $(VENV)/
+	virtualenv -p python $(VENV)/
```

Comment:
This might still be Python 2 on some distros, better switch to `python3` to be explicit.

Also, the Makefile is purely for maintainer convenience, and I don't think keas.kmi has any active maintainers that like Makefiles, so I suggest removing the Makefile entirely.